pom.xml

```diff
@@ -5,7 +5,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>5.2</version>
+        <version>5.9</version>
```

Comment:
Not strictly necessary for this PR, but taking advantage of the build/test cycle to update this plugin's metadata while I am here.